addons/calendar/models/calendar_attendee.py

```diff
@@ -112,7 +112,7 @@ def _send_mail_to_attendees(self, template_xmlid, force_send=False, ignore_recur
 
     def _notify_attendees(self, ics_files, mail_template, rendering_context, force_send):
         for attendee in self:
-            if attendee.email and attendee.partner_id != self.env.user.partner_id:
+            if attendee.email and attendee.partner_id != self.env.user.partner_id and attendee.partner_id.active:
                 # FIXME: is ics_file text or bytes?
                 event_id = attendee.event_id.id
                 ics_file = ics_files.get(event_id)
```

Comment:
Thanks for the quick correction :)

For the commit message. Please add a similar explanation to the one you did in the PR message. 
And this should be a [FIX] tag.

Then I guess it should be fine?

Maybe @arj-odoo can look at this again when he has the time :)

Reply:
@thiryr Thanks bro :)